packages/services/storage/src/index.ts

```diff
@@ -2146,6 +2167,21 @@ export async function createStorage(connection: string, maximumPoolSize: number)
       return output.version;
     },
 
+    async createSchemaSDLChecksum(args) {
+      if (args.schemaSDL === null || args.schemaSDL === undefined) {
+        return null;
+      }
+
+      const checksum = getIntegrityHashOf(args.schemaSDL);
+      const result = await pool.one<schema_sdl_checksums>(sql`
+          INSERT INTO schema_sdl_checksums ("schema_sdl", "checksum", "target_id")
+          VALUES (${args.schemaSDL}, ${checksum}, ${args.targetId}) ON CONFLICT(checksum, target_id)
+          DO UPDATE SET updated_at = NOW()
+          RETURNING *;
```

Comment:
We prefer listing the fields for these return statements instead of returning `*`.

You can do something like

```SQL
RETURNING
  "checksum",
  to_json("created_at") as "createdAt",
  to_json("updated_at") as "updatedAt",
  "target_id" as "targetId",
  "schema_sdl" as "schemaSdl"
```